### PR TITLE
Testimony style proposition

### PR DESCRIPTION
**Pull request by RaphaelBS-WCS**
Style proposition for testimony view:
- carousel +  add button on the section with view height
- section padding-top
submitted to your validation

### CODE REVIEW

**Review by khadimb — APPROVED** [review]
nice

File: src/View/Visitor/Home/Component/_carousel.html.twig [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+                    </p>
+                    <h3 class="testifying">Claire.B</h3>                      
+                </div>
             </div>
         </div>
     </div>
+    <div class="testimony-contact">
+        <a href="#" class="button-testimony">Témoignez en cliquant ici</a>
+    </div>
 </section>
```
Comment (LucyMerlier) [inline src/View/Visitor/Home/Component/_carousel.html.twig:32]: you modified this page in another PR

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

File: src/View/Visitor/Home/Component/_carousel.html.twig (hunk 1 above)
Comment (RaphaelBS-WCS) [reply src/View/Visitor/Home/Component/_carousel.html.twig:32]: yes, but not fot the same reason and for the same lines.

Comment (RaphaelBS-WCS) [reply src/View/Visitor/Home/Component/_carousel.html.twig:32]: we have changes our minds for the style of this section.

**Review by khadimb — DISMISSED** [review]
change ok

**Review by LucyMerlier — APPROVED** [review]

**Review by khadimb — APPROVED** [review]
ok

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**